reconcile/prometheus_rules_tester.py

```diff
@@ -275,6 +306,8 @@ def check_prometheus_tests(tests, rules, thread_pool_size, settings):
     for path, test in tests.items():
         test_to_run = {"path": path}
 
+        target_clusters = get_target_clusters_from_jinja_test_template(test)
+
```

Comment:
can we make something nicer to avoid parsing the same file twice? e.g. a function that returns both the target clusters and the rule files?